src/solvers/dg/3d/amr.jl

```diff
@@ -314,6 +314,68 @@ function calc_amr_indicator(dg::Dg3D, mesh::TreeMesh, time::Float64)
         lambda[element_id] = 0.0
       end
     end
+  elseif dg.amr_indicator === :blob
+    base_level = 1
+    max_level = 6                                                                                   
+    blending_factor_threshold1 = 0.3 # Löhner original choice
+    blending_factor_threshold2 = 0.1 # Löhner original choice
+                                                                                                    
+    # (Re-)initialize element variable storage for blending factor                                  
+    if (!haskey(dg.element_variables, :amr_indicator_values) ||                                     
+        length(dg.element_variables[:amr_indicator_values]) != dg.n_elements)                       
+      dg.element_variables[:amr_indicator_values] = Vector{Float64}(undef, dg.n_elements)           
+    end                                                                                             
+    if (!haskey(dg.element_variables, :amr_indicator_values_tmp) ||                                 
+        length(dg.element_variables[:amr_indicator_values_tmp]) != dg.n_elements)                   
+      dg.element_variables[:amr_indicator_values_tmp] = Vector{Float64}(undef, dg.n_elements)       
+    end                                                                                             
+                                                                                                    
+    alpha     = dg.element_variables[:amr_indicator_values]                                         
+    alpha_tmp = dg.element_variables[:amr_indicator_values_tmp]                                     
+    calc_loehner_indicator!(alpha, alpha_tmp, dg.elements.u, dg.amr_alpha_max, dg.amr_alpha_min, false,
```

Comment:
Why not `calc_löhner_indicator!`? :innocent:  Or are we not allow to use `ö` in code?

Reply:
didn't know ö is allowed 👻

Reply:
hm, but might be bad for people without German keyboard to call this function?

Reply:
Might be. On the other hand, I also don't have a German keyboard :sweat_smile:

Reply:
But you can also leave it as it is.